newdle/client/src/components/creation/FinalStep.js

```diff
@@ -65,31 +87,29 @@ export default function FinalStep() {
           }
         }}
       />
-      <div className={styles['attention-message']}>
-        <Header as="h3" className={styles['header']}>
-          <Trans>Attention</Trans>
-        </Header>
-        {participants.length !== 0 ? (
-          <p>
-            <Trans>
+      {!isEditing && (
+        <div className={styles['attention-message']}>
+          <Header as="h3" className={styles['header']}>
+            <Trans>Attention</Trans>
+          </Header>
+          {participants.length !== 0 ? (
+            <Trans render="p">
               Your participants will receive an e-mail asking them to register to their preference.
               Once the newdle is created, you will be shown a link you can share with anyone else
               you wish to invite.
             </Trans>
-          </p>
-        ) : (
-          <p>
-            <Trans>
+          ) : (
+            <Trans render="p">
               Once the newdle is created, you will be shown a link which you need to send to anyone
               you wish to invite.
             </Trans>
-          </p>
-        )}
-      </div>
+          )}
+        </div>
+      )}
       <div className={styles['advanced-options']}>
         <div
           className={styles['headerbar']}
-          onClick={() => setShowAdvancedOptions(!showAdvancedOptions)}
+          onClick={() => setShowAdvancedOptions(isEditing || !showAdvancedOptions)}
```

Comment:
This prevents you from closing the options. I think the initial value for the `showAdvancedOptions` state is enough and we can simply revert the change in this line.

Reply:
Alternatively, hide the button and pointer cursor and don't make it collapsible in edit mode

Reply:
I thought it wasn't supposed to be collapsible, but the caret is misleading. So perhaps an initial value is the simplest.